source/sfp/1_water/sfp_114_nl_waterstaalname_plas/07_werkwijze.Rmd

```diff
@@ -0,0 +1,77 @@
+Zo kan je hiermee rekening houden bij de staalname.
+Neem bij elke staalname en/of veldmeting steeds een overzichtsfoto en eventueel detailfoto's zodat dit later kan worden geraadpleegd.
+
+### Voorbereiding
+
+Bij aanvang van het veldwerk wordt het materiaal dat in contact komt met het staal overvloedig gespoeld met water van de staalnamelocatie.
+Spoelen met lokaal water helpt het materiaal te acclimatiseren aan het staal en beperkt de residuen van producten of vorige stalen.
+Doe dit zowel voor de recipiënten (uitgez. recipiënt nummer 7), staalname-emmer, maatbeker, spuit, latex handschoenen en laarzen als voor de elektrodes van de multimeter.
+Indien recipiënten gecertificeerd zijn als 'contaminant-vrij' is het spoelen van de recipiënten niet noodzakelijk.
+
+### Nemen van een mengstaal
+
+Kies eerst een geschikte plaats voor de staalname, op of in de directe omgeving (zo dicht mogelijk) van het staalnamepunt.
+Neem geen (deel)staal op plekken die recent werden verstoord door dieren of mensen.
+Ganzen, vee, etc. kunnen door verstoring van de waterbodem en de vegetatie, of door hun fecaliën, of het inbrengen van locatievreemd materiaal het staal contamineren.
+Neem indien mogelijk geen (deel)staal tussen dichte vegetatie en hou voldoende afstand van in- en uitstroompunten van waterlopen, duikers of lozingsvoorzieningen, tenzij dit specifiek wenselijk is.
+Wees alert voor plaatselijke contaminanten bij locaties naast wegen (bijvoorbeeld afstroom van regenwater van weginfrastructuur, strooizout in de winter, etc.) en andere antropogene elementen.
+
+Maak een mengstaal door met de maatbeker op drie punten (bij voorkeur twee of drie meter uit elkaar) een gelijk volume deelstaal (twee liter) te nemen.
+Doe dit steeds voorzichtig zonder de waterbodem of vegetatie te veel te verstoren.
+Laat de maatbeker diep genoeg zakken zodat het deelstaal water van de waterkolom bevat en niet enkel het water aan het oppervlak.
+Giet de inhoud van de maatbeker voorzichtig in de staalname-emmer zonder luchtbellen te veroorzaken.
+
+Indien het nemen van een staal gehinderd wordt door ijsvorming wordt eerst een voldoende grote opening in het ijs gemaakt waaruit alle brokken ijs en eventueel vuil en slib worden verwijderd.
+Vervolgens wacht men enkele minuten zodat het water vrij onder het ijs kan bewegen.
+Om contaminatie van omgewoelde bodem te vermijden wordt enkel bemonsterd als er minstens circa 10 cm water aanwezig is onder de ijslaag (VMM 2020).
+Deze werkwijze mag enkel uitgevoerd worden wanneer de veiligheid van de staalnemer gegarandeerd kan worden.
+
+### Vullen van recipiënten voor het laboratorium
+
+1.  [*Vul het recipiënt voor IPC met spuit en filters*]{.underline}
```

Comment:
deze titel moet niet schuin staan; idem opmaak als lijn 49

Reply:
Opgelost in [9ce87d5](https://github.com/inbo/protocolsource/pull/107/commits/9ce87d50dcf1e11f9e578fc8ab50060155bb5896)